Assets/Scripts/Effects/AnimateWater.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Effects
+{
+    public class AnimateWater : MonoBehaviour
+    {
+        [SerializeField] [Tooltip("Determines the difference between the peak and bottom of a wave")]
+        private float power = 3f;
+    
+        [SerializeField] [Tooltip("Determines the size of waves")]
+        private float scale = 1f;
+    
+        [SerializeField] [Tooltip("Determines how fast waves aree")]
+        private float frequency = 1f;
+
+        private float _xOffset, _yOffset;
+        private MeshFilter _mf;
```

Comment:
Please try to use the most obvious name for variables --> _meshfilter

Reply:
done.